tests/api/test_views.py

```diff
@@ -582,6 +620,132 @@ def test_post_enterprise_customer_user_logged_out(self):
         response = self.client.post(settings.TEST_SERVER + reverse('enterprise-learner-list'), data=data)
         assert response.status_code == 401
 
+    @ddt.data(
+        (
+            'd2098bfb-2c78-44f1-9eb2-b94475356a3f',
```

Comment:
@brittneyexline Nit: Introduce a constant (such as `DEFAULT_ENTERPRISE_UUID`) for this, perhaps?